components/atoms/Select/custom-select.tsx

```diff
@@ -10,72 +13,55 @@ interface SelectProps {
   className?: string;
 }
 
-type SelectedState = {
-  name: string;
-  value: any;
-};
-const Select = ({
+const CustomSelect = ({
   placeholder = "Select an option",
   options,
-  error, // don't know if this is necessary
   label,
   className,
   onChange
 }: SelectProps): JSX.Element => {
-  const [isOpen, setIsOpen] = useState<boolean>(false);
-  const [selected, setSelected] = useState<SelectedState>({ name: "", value: "" });
-  const handleToggle = () => {
-    setIsOpen(!isOpen);
-  };
-  const handleSelected = (option: { name: string; value: any }) => {
-    const { name, value } = option;
-    setSelected((prev) => ({ ...prev, name, value }));
-    onChange?.(value);
-    setIsOpen(false);
+  const handleSelected = (value: string) => {
+    const selected = Number(value.substring(0, 2));
```

Comment:
We should be using the actual value instead of a substring of the name. The `options` should provide an array of name/value objects

```js
[
  { name: '10 per page', value: 10 }
]
```